src/Fieldtypes/IconPicker.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Statamic\Fieldtypes;
+
+use Statamic\Facades\Folder;
+use Statamic\Fields\Fieldtype;
+
+class IconPicker extends Fieldtype
+{
+    protected $categories = ['media'];
+    protected $selectable = false;
+
+    public function preload(): array
+    {
+        $folder = $this->config('folder', 'resources/svg');
```

Comment:
We should probably allow any path instead of just a folder. Then the default could be `statamic_path('resources/svg')`. That should let someone point to their own icons.